tests/unit/s2n_security_policies_test.c

```diff
@@ -921,14 +958,49 @@ int main(int argc, char **argv)
                 &security_policy_20240502,
             };
 
-            const struct s2n_supported_cert supported_certs[] = {
-                { .cert = rsa_chain_and_key },
-                { .cert = ecdsa_chain_and_key },
-            };
+            DEFER_CLEANUP(struct s2n_test_cert_chain_list cert_chains = { 0 },
+                    s2n_test_cert_chains_free);
+            EXPECT_OK(s2n_test_cert_chains_init(&cert_chains));
+            EXPECT_OK(s2n_test_cert_chains_set_supported(&cert_chains,
+                    S2N_PKEY_TYPE_RSA_PSS, S2N_TEST_CERT_UNSUPPORTED));
+            EXPECT_OK(s2n_test_cert_chains_set_supported(&cert_chains,
+                    S2N_PKEY_TYPE_MLDSA, S2N_TEST_CERT_UNSUPPORTED));
 
             EXPECT_OK(s2n_test_default_backwards_compatible("default_fips",
                     versioned_policies, s2n_array_len(versioned_policies),
-                    supported_certs, s2n_array_len(supported_certs)));
+                    &cert_chains));
+        };
+
+        /* "default_pq" */
+        {
+            const struct s2n_security_policy *versioned_policies[] = {
```

Comment:
Is there a reason `security_policy_20241001` was omitted?

Reply:
Oh I see it in the history: https://github.com/aws/s2n-tls/commit/645c3743a7cfdaa84a18bdda046d53470956c73c
I guess we just missed that one in the documentation?

Reply:
Well I'm glad you couldn't find the documentation, because otherwise you wouldn't have caught that old mistake xD